src/Accelerators/InitAccelerators.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===------------------------ InitAccelerators.cpp ------------------------===//
+//
+// Copyright 2022 The IBM Research Authors.
+//
+// =============================================================================
+//
+// Initialization of accelerators.
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/Accelerators/Accelerator.hpp"
+
+namespace onnx_mlir {
+namespace accel {
+
+//
+bool initAccelerators() {
+  //#ifdef HAS_ACCELERATORS
+  APPLY_TO_ACCELERATORS(DECLARE_ACCEL_INIT_FUNCTION)
```

Comment:
I think the declarations don't have to be inside the function

Reply:
ok will move it